src/Rec/Rec.spec.ts

```diff
@@ -353,6 +419,12 @@ describe("Rec", () => {
     })
 
     describe("remove", () => {
+        it("ignores object keys that were manually set to undefined", () => {
+            expect(pipe({ a: undefined }, Rec.remove("a"))).toStrictEqual({
+                a: undefined,
+            })
```

Comment:
⚠️ This feels inconsistent with the behavior of `Rec.map`  where undefined gets removed instead of passed through. I'm wondering if the behavior in `Rec.map` is the one that should change. I.e. it should pass through `undefined` but not run the passed in `f` against it. Thoughts?

Reply:
So, in this implementation, I was thinking in terms of, "What is the logical inverse of the `set` operation?" For the `set` function, `undefined` entries will persist, so I thought they should here too with `remove`.

I guess conceptually I view `map` differently as it is a "operate on the entire collection of entries" rather than a "only care about a single entry" kind of operation. Thoughts?

Reply:
I think I agree with you but I'm just thinking from a "strict equals" perspective like this test. Would this test still pass if it was `.toStrictEqual({})` ?

Reply:
Hmph. That's a good point 🤔 I'm not sure there is a "right" answer here, just different ones lool. I think the "safe" thing to do though is remove the key anyway, like you said...... yeah, I think I'm gonna do that. It shouldn't make a difference to an API consumer, and makes it slightly easier to reason about the runtime shape of the object

Reply:
:+1:

Reply:
https://github.com/fp-toolkit/fp-toolkit/pull/69/commits/c74fd4295f7ed4bb0f90a47c382ceec3f2bd506d

Reply:
Can you peep the improved implementation logic to make sure it's sane?

Reply:
Looks good to me! Just had one small 🎨 consideration.